### PR TITLE
added sql reference for SHOW commands

### PR DESCRIPTION
**Pull request by mikethebeer**
* show tables
* show columns
* show schemas

### CODE REVIEW

File: docs/sql/reference/show_schemas.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+============
+
+Lists the table schemas of the database.
+
+Synopsis
+========
+
+::
+
+    SHOW SCHEMAS [LIKE 'pattern']
```
Comment (chaudum) [inline docs/sql/reference/show_schemas.txt]: SHOW SCHEMAS [ LIKE 'pattern' | WHERE expression ]

File: docs/sql/reference/show_columns.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+``SHOW COLUMNS`` fetches all column names of a given table and displays their 
+detailed information. The column names get listed in alphabetical order.
+
+This command is also applicable on SYSTEM tables (see :ref:`system-information`) 
+and BLOB tables (see :ref:`blob_support`).
+
+Parameters
+==========
+
+:table_name:  The name of the table that should print its column information.
```
Comment (chaudum) [inline docs/sql/reference/show_columns.txt]: The name of the table of which the column information is printed.

File: docs/sql/reference/show_columns.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+::
+
+    SHOW COLUMNS {FROM | IN} table_name [FROM | IN schema_name] [LIKE 'pattern']
+
+Description
+===========
+
+``SHOW COLUMNS`` fetches all column names of a given table and displays their 
+detailed information. The column names get listed in alphabetical order.
```
Comment (chaudum) [inline docs/sql/reference/show_columns.txt]: **are** listed ...

File: docs/sql/reference/show_columns.txt [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+This command is also applicable on SYSTEM tables (see :ref:`system-information`)
```
Comment (chaudum) [inline docs/sql/reference/show_columns.txt]: system tables

File: docs/sql/reference/show_schemas.txt [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Description
+===========
+
+``SHOW SCHEMAS`` can be used to retrieve defined schema names of the database in 
+alphabetical order. 
+The same list can be fetched by querying the distinct schema names of the
```
Comment (chaudum) [inline docs/sql/reference/show_schemas.txt]: ... the schema names from the `information_schema.schemata` table.

File: docs/sql/reference/show_columns.txt [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+This command is also applicable on system tables (see :ref:`system-information`) 
+and BLOB tables (see :ref:`blob_support`).
+
+Parameters
+==========
+
+:table_name:  The name of the table of which the column information is printed.
+              If no schema name is specified the default schema is set to 
+              ``doc``.
```
Comment (dobe) [inline docs/sql/reference/show_columns.txt]: this sentence belongs in the next para

File: docs/sql/reference/show_columns.txt [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+LIKE Clause
+===========
+
+The optional ``LIKE`` clause indicates which table names to match. It takes a 
+string pattern as a filter.
+
+.. note::
+
+    Schema names are ignored in the filter that is defined in the ``LIKE`` 
+    clause.
```
Comment (dobe) [inline docs/sql/reference/show_columns.txt]: what kind of pattern is this? probably refer to like if it is the same

File: docs/sql/reference/show_columns.txt [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+::
+
+    SHOW COLUMNS {FROM | IN} table_name [FROM | IN schema_name] [LIKE 'pattern']
+
+Description
+===========
+
+``SHOW COLUMNS`` fetches all column names of a given table and displays their 
+detailed information. The column names are listed in alphabetical order.
+
```
Comment (dobe) [inline docs/sql/reference/show_columns.txt]: refer here also to the information schema like in show schema

File: docs/sql/reference/show_columns.txt [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+This command is also applicable on system tables (see :ref:`system-information`) 
+and BLOB tables (see :ref:`blob_support`).
```
Comment (dobe) [inline docs/sql/reference/show_columns.txt]: imho this note can be removed

File: docs/sql/reference/show_schemas.txt [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+Description
+===========
+
+``SHOW SCHEMAS`` can be used to retrieve defined schema names of the database in 
+alphabetical order. 
+The same list can be fetched by querying the the schema names from the 
+``information_schema.schemata`` table.  
+
+The SYSTEM (see :ref:`system-information`) and BLOB schema 
+(see :ref:`blob_support`) are included in the list.
```
Comment (dobe) [inline docs/sql/reference/show_schemas.txt]: same here, imho not needed

File: docs/sql/reference/show_schemas.txt [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+LIKE Clause
+===========
+
+The optional ``LIKE`` clause indicates which schema names to match. It takes a 
+string pattern as a filter.
```
Comment (dobe) [inline docs/sql/reference/show_schemas.txt]: try to be consistent, see the other llike descriptions, refer to the LIKE docu

File: docs/sql/reference/show_tables.txt [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+::
+
+    SHOW TABLES [{FROM | IN} schema_name] [LIKE 'pattern']
+
+Description
+===========
+
+``SHOW TABLES`` can be used to retrieve the table names of the database in 
+alphabetical order. The same list can be fetched by querying distinct table 
+names of the :ref:`information_schema` table.
```
Comment (dobe) [inline docs/sql/reference/show_tables.txt]: information_schema.tables table

File: docs/sql/reference/show_tables.txt [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+::
+
+    SHOW TABLES [{FROM | IN} schema_name] [LIKE 'pattern']
+
+Description
+===========
+
+``SHOW TABLES`` can be used to retrieve the table names of the database in 
+alphabetical order. The same list can be fetched by querying distinct table
```
Comment (dobe) [inline docs/sql/reference/show_tables.txt]: why distinct?

File: docs/sql/reference/show_tables.txt [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+Description
+===========
+
+``SHOW TABLES`` can be used to retrieve the table names of the database in 
+alphabetical order. The same list can be fetched by querying distinct table 
+names of the :ref:`information_schema` table.  
+
+Tables of :ref:`system-information` are only listed when they are explicitly 
+specified in ``FROM | IN`` clause. Blob tables cannot be listed at all.
```
Comment (dobe) [inline docs/sql/reference/show_tables.txt]: why?

**Comment by dobe** [comment]
in general, try to be consistent. describe the parameters always in the sama fashion. the params are missing in some cases

File: docs/sql/reference/show_schemas.txt [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+The optional ``LIKE`` clause indicates which schema names to match. It takes a 
+string pattern as a filter and has an equivalent behavior to 
+:ref:`sql_dql_like`.
+
+WHERE Clause
+============
+
+This clause is optional and has the same implementation as 
+:ref:`sql_dql_where_clause`.
```
Comment (dobe) [inline docs/sql/reference/show_schemas.txt]: refer to information_schema.schemas

**Comment by dobe** [comment]
ack

**Pull request merged**